src/styles/blocks/main.scss

```
@@ -0,0 +1,15 @@
+.main {
+  padding: 50px 128px;
```

```suggestion
  padding: 50px;
```
try to use flex, not padding